sugaroid/brain/GuessTheNumber.py

```diff
@@ -0,0 +1,65 @@
+import random
+from sugaroid.brain.constants import HOPE_GAME_WAS_GOOD
+from sugaroid.brain.postprocessor import random_response
+from sugaroid.brain.ooo import Emotion
+from sugaroid.core.base_adapters import SugaroidLogicAdapter
+from sugaroid.core.statement import SugaroidStatement
+
+GUESS_NUM_WIN = [
+"Congragulations! You really are a genius.",
+"You havae won the game by Guessing the Number correctly.",
+"Let the celebrations begin!",
+]
+
+GUESS_NUM_LOS = [
+"That was close.",
+"Better Luck Next Time Mate.",
+"Be enthusiastic and give it a try once more!',
```

Comment:
Kindly close the statement with the same double quote you've used at the start..